### ISSUE TITLE
Docker container is not building anymore

### ISSUE THREAD

**Issue by mwalzer** [FRED-2/OptiType#65]
hdf5 resources are inavailable

**Comment by mwalzer** [comment]
maybe building from https://hub.docker.com/r/hdfgroup/hdf5lib/ would work?

**Comment by b-schubert** [comment]
@mwalzer could you try and build the container again. The container hosted at docker hub build correctly.

**Comment by mwalzer** [comment]
Yeah, now https://support.hdfgroup.org/ftp/HDF5/current18/bin/linux-centos7-x86_64-gcc485/hdf5-1.8.19-linux-centos7-x86_64-gcc485-shared.tar.gz is available again.
Still, I think to build from a docker image might be the more stable option.

**Comment by b-schubert** [comment]
Agreed. Will change the Dockerfile as soon as I find time.

**Comment by allteaser** [comment]
running docker, the following error occurs:
File "/usr/local/bin/OptiType/OptiTypePipeline.py", line 11
SyntaxError: Non-ASCII character '\xc3' in file /usr/local/bin/OptiType/OptiTypePipeline.py on line 12, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details

**Comment by allteaser** [comment]
It works when pulled from here
